### ISSUE TITLE
Cloning Starts From Beginning if it stops ! its a mess ! need fix pls

### ISSUE THREAD

**Issue by MHJoy99** [Grey-Time/Telethon-Channel-Clone#2]
I Mean when we start cloning a channel if by chance we stop it then it starts from the begining ! i need fix for that! help pls

**Comment by G4latea** [comment]
> I Mean when we start cloning a channel if by chance we stop it then it starts from the begining ! i need fix for that! help pls
I agree. The app clone like 2k or 3k at first, then there is a cooldown of 3000 segs. Then it clones another 2k messages then another cooldown fo 3000 segs. So if you have a larger group, you need to beg that this not stop or else you need to start from the begging. SO i do not know if there is a way to introduce the exact number of the current message to the config.py file or somewhere in the library of the app so the it can keep from the current.
